docs/alerts/understanding-alerts.md

```diff
@@ -0,0 +1,32 @@
+# 1. Understanding Alerts in Netdata
+
+In Netdata, an **alert** (also called a "health check") is a rule that continuously monitors one or more metrics and assigns a **status** based on whether conditions are met.
+
+| Core Concept | Description |
+|--------------|-------------|
+| **Where alerts are evaluated** | Locally on each Netdata Agent or Parent node, not centrally in Netdata Cloud |
+| **How alerts work** | Each alert inspects recent metric data (for example, "average CPU over the last 5 minutes") and decides whether the system is in a healthy state (`CLEAR`), needs attention (`WARNING`), or has a problem (`CRITICAL`) |
+| **What happens on status change** | When an alert's status changes, that transition becomes an **alert event** visible in the Agent dashboard, APIs, and Netdata Cloud's Events Feed |
+
+## Where Alerts Run
+
+- **Agents** evaluate alerts against their own collected metrics
+- **Parents** (in streaming setups) can evaluate alerts on metrics received from child nodes
+- **Netdata Cloud** receives alert events from Agents/Parents and presents them in a unified view, but does **not** re-evaluate the rules itself
```

Comment:
I'd say alert events are received from Agents (without calling out Parents), but then we probably should mention that if multiple Agents represent a given node, their reported statuses are consolidated for that node.